README.rst

```diff
@@ -140,7 +140,7 @@ This is a non-exhaustive list of MPD features that Mopidy doesn't support.
 - Replay gain is not supported.
 - ``stats`` does not provide any statistics.
 - ``decoders`` does not provide information about available decoders.
-- ``tagtypes`` is not supported.
+- ``tagtypes`` is not fully supported.
```

Comment:
can this line be removed now? if not, what functionality is still missing? (i'm not saying you should implement that too, just want to know what this is about)

Reply:
Good catch, when I first started working on this I had just stubbed out the subcommands when I changed that. As far as I know it should be fully supported now so I removed that line.